index.html

```diff
@@ -76,13 +83,13 @@ <h1>
     </section>
     <section class="recent-works">
       <div class="section-heading flex">
-        <h1>My Recent<br>Works</h1>
+        <h1>My Recent Works</h1>
         <div class="indicator"></div>
       </div>
       <div class="card-container">
         <div class="card flex">
           <img
-            src="https://images.unsplash.com/photo-1618477247222-acbdb0e159b3?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=700&q=80"
+            src="https://images.unsplash.com/photo-1618477247222-acbdb0e159b3/?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=700&q=80"
             class="image" alt="placeholder" />
           <div class="inner-box flex">

```

Comment:
- [ ] It seems that the recent works section does not look like the one in the Figma design yet, see the difference below 👇 
- Your design
![image](https://user-images.githubusercontent.com/55465133/129945344-80ca94b6-8d22-409f-a259-d6fa41b638d0.png)

- Figma design
![image](https://user-images.githubusercontent.com/55465133/129945406-1b9ee7e1-e1d5-4cf3-9c11-eedeff18275b.png)

we can see that in the Figma design we have 3 cards aligned in each row, but for you, we only see 2 cards, please let us use the `grid` layout to render properly these cards 😉

Reply:
Thank you. I think you are right I'll adjust the sizing of the the grid to compensate for this.